config/initializers/geocoder.rb

```diff
@@ -0,0 +1,22 @@
+Geocoder.configure(
+  # Geocoding options
+  # timeout: 3,                 # geocoding service timeout (secs)
+  lookup: :google,            # name of geocoding service (symbol)
+  # ip_lookup: :freegeoip,      # name of IP address geocoding service (symbol)
+  # language: :en,              # ISO-639 language code
+  use_https: true,           # use HTTPS for lookup requests? (if supported)
+  # http_proxy: nil,            # HTTP proxy server (user:pass@host:port)
+  # https_proxy: nil,           # HTTPS proxy server (user:pass@host:port)
+ api_key: "AIzaSyCmoYC2frxheJfdqWCa3kvb6SMlT7ThTaU" ,               # API key for geocoding service
```

Comment:
Remember here we need to replace this with a universal environment variable and we now need a new API key